### PR TITLE
Us11 mohamed

### PR DESCRIPTION
**Pull request by Raymond352**
Finished my test cases.
Update configuration.properties "browser=chrome"

### CODE REVIEW

**Comment by Raymond352** [comment]
I also added more time for closing the browser it was closing so fast for me.

File: src/test/java/com/danica_intabella/step_definitions/Hooks.java
Comment (Saharnaz1) [inline src/test/java/com/danica_intabella/step_definitions/Hooks.java:1]: Are we supposed to use thread.sleep here and not explicit wait?

File: configuration.properties
Comment (Saharnaz1) [inline configuration.properties:1]: We have to ask Alvin, the test cases we did using headless passed. I have to see if others have an issue

File: src/test/resources/features/US11_VehicleOdometer.feature
Comment (Saharnaz1) [inline src/test/resources/features/US11_VehicleOdometer.feature:1]: I will wait until Alvin reviews this if he hasn't already before merging

File: src/test/java/com/danica_intabella/step_definitions/Hooks.java
Comment (Raymond352) [reply src/test/java/com/danica_intabella/step_definitions/Hooks.java:1]: Interested question we'll go over it in the meeting.

File: configuration.properties
Comment (Raymond352) [reply configuration.properties:1]: I think we'll only need to change it in configuration.prperties

File: src/test/resources/features/US11_VehicleOdometer.feature
Comment (Raymond352) [reply src/test/resources/features/US11_VehicleOdometer.feature:1]: Yes, sure.
Thank you though for checking

**Review by PuujeeS — APPROVED** [review]
I reviewed.

**Review by Alvin253535 — APPROVED** [review]
good job

**Pull request merged**
